charts/portworx/v1.0.0/questions.yml

```diff
@@ -160,7 +160,7 @@ questions:
   type: string
   group: "Advanced parameters"
 - variable: imageVersion
-  default: "1.5.1"
+  default: "1.7.1.1"
```

Comment:
shouldn't the version be `1.7.1` instead of `1.7.1.1`? 
and basically we are not recommend to update the existing chart without bump the chart version.

Reply:
@guangbochen for the 1.0.0 chart version, I've reverted the versions.

So only the latest chart (1.0.1) has the new image versions.

Also the version should be 1.7.1.1 since that's the Portworx release version when the spec is installed using https://install.portworx.com. Portworx release notes can be found at: https://docs.portworx.com/reference/release-notes/#errata